### ISSUE TITLE
Disabling HiDPI interface scaling doesn't work

### ISSUE THREAD

**Issue by dominalien** [linuxmint/mint20-beta#106]
1080p 13" screen, Intel Core i7-7200U with HD Graphics 620.
In screen properties I have zoom set to 125% and interface scaling set to HiDPI. When I set interface scaling to normal and press apply, I get a very tall (multiple screens) error window like shown in the attached screenshot. After pressing enter (I scrolled the window once and it just has one Close button at the bottom) Cinnamon crashes.
![Zrzut ekranu z 2020-06-18 08-31-18](https://user-images.githubusercontent.com/37808252/84986322-ac002200-b13e-11ea-85d2-731e910f026d.png)

**Comment by dominalien** [comment]
Setting scaling to 100% resets interface scaling to normal and then clicking apply works.

**Comment by mtwebster** [comment]
Please checkout this comment:
https://github.com/linuxmint/cinnamon-control-center/pull/238#issuecomment-646734935

Thanks

**Comment by dominalien** [comment]
It's a bit different, but the problem remains. Once I got this window when enabling HiDPI interface scaling, but cinnamon didn't crash. Once I got both 125% and HiDPI enabled, disabling HiDPI caused the error and cinnamon crash. Setting zoom to 100% no longer reset HiDPI to normal Logging out and back in allowed me to get back to 100% and normal scaling.

Performance with 125% and HiDPI is still dire and there's plenty of tearing.
